src/main/java/graphql/execution/instrumentation/dataloader/FieldLevelTrackingApproach.java

```diff
@@ -181,6 +182,32 @@ private int getCountForList(List<FieldValueInfo> fieldValueInfos) {
         return result;
     }
 
+    InstrumentationContext<ExecutionResult> beginDeferredField(InstrumentationDeferredFieldParameters parameters, InstrumentationState state) {
```

Comment:
consider removing support for data loaders with defer - throw an exception - we'll implement later